client/cypress/integration/ignoredTestFiles/route_tests.spec.js

```diff
@@ -31,7 +273,9 @@ describe("Route tests", () => {
             `http://localhost:8080/build/InfoBase/index-${app}.html#${routes.route}`
           );
           cy.injectAxe();
-          cy.get(".leaf-spinner-inner-circle").should("not.exist");
+          // eslint-disable-next-line cypress/no-unnecessary-waiting
+          cy.wait(5000);
```

Comment:
This "works" but was trying to avoid as per: https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting. Since multiple routes have a number of different things being loaded after the initial page load, we need to wait longer to assert that the spinner is no longer present, so maybe it's an okay rule break? But this is why I've been trying other methods of delaying the spinner assertion such as intercepting some requests.

Reply:
Just now noticing the wait is at the wrong place, should be after `cy.visit()` command. Mix up happened after some clean up before committing... but concept doesn't change. The file is being ignored from CI for now regardless.